src/ray/gcs/store_client/in_memory_store_client.h

```diff
@@ -33,43 +33,44 @@ namespace ray::gcs {
 /// This class is thread safe.
 class InMemoryStoreClient : public StoreClient {
  public:
-  explicit InMemoryStoreClient(instrumented_io_context &main_io_service)
-      : main_io_service_(main_io_service) {}
+  explicit InMemoryStoreClient() = default;
 
   Status AsyncPut(const std::string &table_name,
                   const std::string &key,
                   std::string data,
                   bool overwrite,
-                  std::function<void(bool)> callback) override;
+                  Postable<void(bool)> callback) override;
 
   Status AsyncGet(const std::string &table_name,
                   const std::string &key,
-                  OptionalItemCallback<std::string> callback) override;
+                  ToPostable<OptionalItemCallback<std::string>> callback) override;
```

Comment:
```suggestion
                  Postable<OptionalItemCallback<std::string>> callback) override;
```

?

Reply:
`ToPostable<std::function<Ret(Args...)>>` is a type == `Postable<Ret(Args...)>`. I use this to convert familiar Ray Callback types to make sure the change is minimal.